tests/phpunit/tests/formatting/excerptRemoveFootnotes.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * @group formatting
+ * @ticket 58805
+ *
+ * @covers ::excerpt_remove_footnotes
+ */
+
+class Tests_Formatting_ExcerptRemoveFootnotes extends WP_UnitTestCase {
+	/**
+	 * @ticket 58805
+	 *
+	 * @dataProvider data_remove_footnotes
+	 *
+	 * @param string $expected  Expected output.
+	 * @param string $content   Content to run strip_shortcodes() on.
+	 * @param string $message   The test message.
+	 */
+	public function test_remove_footnotes( $expected, $content, $message = '' ) {
```

Comment:
```suggestion
	public function test_remove_footnotes( $expected, $content, $message ) {
```
We have message for all the cases that why it assign null?

Reply:
Can I blame Code Pilot :trollface: